utils/run.js

```diff
@@ -1,29 +1,37 @@
 const { execSync, spawn } = require('child_process');
 const { join, relative, sep } = require('path');
-const { readdirSync } = require('fs');
 
-if (
```

Comment:
How come this logic is removed?

Reply:
It was only necessary when we would deploy forks during testing but we haven't done that for awhile since changing the token factory to the registration api.

I can add it back if you think its better to show green when someone is testing in their local fork?

Reply:
Ok makes sense.